llvm/include/llvm/Support/BinaryStreamWriter.h

```diff
@@ -109,15 +109,15 @@ class BinaryStreamWriter {
   ///
   /// \returns a success error code if the data was successfully written,
   /// otherwise returns an appropriate error code.
-  Error writeStreamRef(BinaryStreamRef Ref);
+  Error writeStreamRef(const BinaryStreamRef &Ref);
```

Comment:
The Ref seems to indicate this was intended to be passed by value. I assume this is 16-size? I would expect the minimum threshold to be at least > 16, not >= 16